src/soundfile.js

```diff
@@ -886,6 +886,14 @@ class SoundFile {
   }
 
   // TO DO: document this
+  /**
+   *  Set the pitch rate of a sound file. Will change the pitch and the speed also.
+   *
+   *  @method setPitch
+   *  @for p5.SoundFile
+   *  @param {Number} pitchRate     Set the pitch rate.
+   *                                Values ​​less than zero are also accepted.
```

Comment:
I think we probably shouldn't mention negative values here, since this method takes MIDI notes as inputs, and MIDI doesn't define negative notes, so it might confuse people. [The docs for `rate()` already mention backwards playback](https://p5js.org/reference/#/p5.SoundFile/rate) so people can use this method for that purpose.

Reply:
then we should delete line number 895?

Reply:
I think so, for this method.